### ISSUE TITLE
Indentation rule reports incorrect errors with Sass maps

### ISSUE THREAD

**Issue by maxlapides** [sasstools/sass-lint#104]
Using this indentation rule:

```
  indentation:
    - 2
    -
      size: 4
```

I have a Sass map that looks like this (starting on line 1 of my file):

```scss
$colors: (
    base: (
        red: #fff,
        blue: #fff
    ),
    text: (
        default: #fff,
        light: #fff
    )
);
```

And here's the error I get:

```
app/scss/_colors.scss
  6:5  error  Indentation of 4, expected 8   indentation
  7:9  error  Indentation of 8, expected 12  indentation
  8:9  error  Indentation of 8, expected 12  indentation
```

Thanks for any help :)

**Comment by DanPurdy** [comment]
Yeah, recreated this. Will look into it.

it's currently expecting to see the following which is clearly wrong.

```scss
$colors: (
    base: (
        red: #fff,
        blue: #fff
    ),
        text: (
            default: #fff,
            light: #fff
    )
);
```

**Comment by DanPurdy** [comment]
Relates also to #35
